docs/content/en/docs/packages/packagebundles.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Managing package bundles"
+linkTitle: "Package bundles"
+weight: 5
+aliases:
+    /docs/tasks/packages/packagebundles/
+---
+
+### Getting new package bundles
+Package bundle resources created and managed in the management cluster, so first set up `KUBECONFIG` environment variable for the management cluster.
```

Comment:
```suggestion
Package bundle resources are created and managed in the management cluster, so first set up `KUBECONFIG` environment variable for the management cluster.
```

Reply:
is there a place explaining what package bundle is?

Reply:
@jiayiwang7 It is mentioned on the [overview page](https://anywhere.eks.amazonaws.com/docs/packages/overview/). I vote for making improvements to it, possibly in a next PR.

Reply:
cool, once we have a better description of what package bundle is in overview, can we link it here as well?